components/gauge/guage.css

```diff
@@ -89,16 +86,15 @@
   height: 30px;
   background-color: #000;
   -webkit-transform: translate(-50%, -50%);
-          transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
 }
 .analyze--gauge tbody tr td .analyze-data {
   position: absolute;
   left: calc(var(--value, 0) * 100%);
   top: 1rem;
   -webkit-transform: translateX(-50%);
-          transform: translateX(-50%);
+  transform: translateX(-50%);
   white-space: nowrap;
   font-weight: bold;
   color: #000;
-}
-/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uL3NyYy9jaGFydHMvX2dhdWdlLnNjc3MiLCJjaGFydHMuY3NzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBOztFQUFBO0FBZ0JBO0VBQ0Usd0JBQUE7RUFDQSxjQUFBO0VBQ0Esb0JBQUE7RUFBQSxvQkFBQTtFQUFBLGFBQUE7RUFDQSw0QkFBQTtFQUFBLDZCQUFBO01BQUEsMEJBQUE7VUFBQSxzQkFBQTtBQ1pGO0FEY0U7RUFDRSxVQUFBO0VBQ0EsU0FBQTtFQUNBLFlBQUE7RUFDQSx1QkFBQTtBQ1pKO0FEY0U7RUFDRSxvQkFBQTtFQUFBLG9CQUFBO0VBQUEsYUFBQTtFQUNBLFdBQUE7QUNaSjtBRGlCSTtFQUNFLG1CQUFBO01BQUEsV0FBQTtVQUFBLE9BQUE7RUFDQSxrQkFBQTtFQUNBLGlCQUFBO0VBQ0EsbUJBQUE7QUNYTjtBRGFJO0VBQ0UsZ0JBQUE7QUNYTjtBRGFJO0VBQ0Usa0JBQUE7QUNYTjtBRGFJO0VBQ0UsaUJBQUE7QUNYTjtBRGNFO0VBQ0Usb0JBQUE7RUFBQSxvQkFBQTtFQUFBLGFBQUE7RUFDQSx3QkFBQTtNQUFBLHFCQUFBO1VBQUEsdUJBQUE7RUFDQSx5QkFBQTtNQUFBLHNCQUFBO1VBQUEsbUJBQUE7RUFDQSxZQUFBO0VBQ0Esa0JBQUE7RUFDQSxtRUFDQTtFQUNBLHVDQUFBO0FDYko7QURpQk07RUFDRSxrQkFBQTtFQUNBLGlCQUFBO0VBQ0EsUUFBQTtFQUNBLG1DQUFBO1VBQUEsMkJBQUE7RUFDQSxtQkFBQTtBQ1hSO0FEYU07RUFDRSxXQUFBO0VBQ0EsaUJBQUE7RUFDQSxrQkFBQTtBQ1hSO0FEYU07RUFDRSxVQUFBO0VBQ0EsZ0JBQUE7RUFDQSxpQkFBQTtBQ1hSO0FEYU07RUFDRSxrQkFBQTtFQUNBLFdBQUE7RUFDQSxXQUFBO0VBQ0Esc0JBQUE7QUNYUjtBRFlRO0VBQ0UsV0FBQTtFQUNBLGtCQUFBO0VBQ0Esa0NBQUE7RUFDQSxRQUFBO0VBQ0EsVUFBQTtFQUNBLFlBQUE7RUFDQSxzQkFBQTtFQUNBLHdDQUFBO1VBQUEsZ0NBQUE7QUNWVjtBRFlRO0VBQ0Usa0JBQUE7RUFDQSxrQ0FBQTtFQUNBLFNBQUE7RUFDQSxtQ0FBQTtVQUFBLDJCQUFBO0VBQ0EsbUJBQUE7RUFDQSxpQkFBQTtFQUNBLFdBQUE7QUNWViIsImZpbGUiOiJjaGFydHMuY3NzIn0= */
```

Comment:
Looks like this was previously generated from a `scss` file or similar? Perhaps we should preserve that in the SDC setup? The actual CSS looks like it has things like prefixer etc.

Reply:
There aren't any scss files in the repo so I've no idea where it was generated from: I've removed it to keep the code clean, and it can go back in if the original maintainer regenerates the css however they did before.